server/models/notifications.js

```diff
@@ -1,13 +1,51 @@
 const pool = require('../../db/index.js');
 
 module.exports = {
-  addNotification: async (values = []) => {
-    const query = `INSERT INTO notifications (date, notification, accounts_id) VALUES ($1, $2, $3)`;
+  addNotification: async (values = [], username) => {
+    console.log(values);
+
+
     try {
-      const res = await pool.query(query, values);
-      return res;
+      const query1 = `INSERT INTO notifications (date, title, guests, location, notification) VALUES ($1, $2, $3, $4, $5) RETURNING id;`
+
+      const res1 = await pool.query(query1, values);
+      console.log(res1.rows[0]);
+
+      const query2 = `INSERT INTO accounts_notifications_appointments (notifications_id, accounts_id) VALUES (${res1.rows[0].id}, (SELECT id FROM accounts WHERE username = '${username}'))`
+
+      const res2 = await pool.query(query2);
```

Comment:
It doesn't look like you're using res2 anywhere. This can be edited to `await pool.query(query2);`